.github/workflows/main.yml

```diff
@@ -205,17 +208,34 @@ jobs:
           -DCMAKE_INSTALL_PREFIX="build/distrib" \
           -DPACKAGE_NAME="streamfx-${{ matrix.id }}" \
           -DPACKAGE_PREFIX="build/package" \
-          -DQt5_DIR=/usr/local/opt/qt/lib/cmake/Qt5
+          -DQt5_DIR=/usr/local/Cellar/qt@5/5.15.2/lib/cmake/Qt5
     - name: "Build Project"
       shell: bash
       run: |
         cmake --build "build/temp" --config RelWithDebInfo --target install
+    - name: 'Install prerequisite: Packages app'
+      if: success()
+      shell: bash
+      run: |
+        curl -L -O https://s3-us-west-2.amazonaws.com/obs-nightly/Packages.pkg
```

Comment:
Is it possible to do this without third-party dependencies? _Especially_ third party references to binaries uploaded by OBS Project?

Reply:
I am sure that there is, but I do not know it.